### ISSUE TITLE
"Element Attribute Should Have Value" no longer supports ${None}

### ISSUE THREAD

**Issue by bharkreader** [robotframework/SeleniumLibrary#1708]
## Steps to reproduce the issue

```
*** Settings ***
Library     SeleniumLibrary    timeout=30   implicit_wait=5   run_on_failure=Capture Page Screenshot

*** Test Cases ***
Test
    ${chrome_options} =    Evaluate    sys.modules['selenium.webdriver'].ChromeOptions()    sys
    Call Method    ${chrome_options}    add_argument   --headless
    Create Webdriver  Chrome  my_alias  chrome_options=${chrome_options}  executable_path=/usr/local/bin/chromedriver
    Go To    https://www.google.com
    ${S2L}          Get Library Instance    SeleniumLibrary
    ${WEBDRIVER} =    Set Variable    ${S2L.driver}

    Element Attribute Value Should Be    //img[contains(@alt, "Google")]    madeupattr    ${None}
```

```
==============================================================================
Test
==============================================================================
Test                                                                  | FAIL |
Element '//img[contains(@alt, "Google")]' attribute should have value 'None' but its value was 'None'.
------------------------------------------------------------------------------
Test                                                                  | FAIL |
1 test, 0 passed, 1 failed
==============================================================================
Output:  /Volumes/Workspace/git/fe/q2smart_automation/output.xml
Log:     /Volumes/Workspace/git/fe/q2smart_automation/log.html
Report:  /Volumes/Workspace/git/fe/q2smart_automation/report.html
```

With some manual testing, it looks like providing the str hint is converting the none to string. I think we should use `Union[None, str]` instead of just str for the expected argument. 

## Expected behavior and actual behavior

I expect this test to pass because Get Element Attribute returns ${None}. I have tests that follow this pattern that have started failing after the latest robotframework-seleinum update.

## Environment

Browser: Chrome Version 89.0.4389.90 (Official Build) (x86_64)
Browser driver: headless chrome=89.0.4389.90
Operating System: Mac 10.14.6
Libraries
- Robot Framework: Version 4.0
- Selenium: Version 3.141.0
- SeleniumLibrary: Version 5.1.1
- Interpreter: Python 3.7.6

**Comment by aaltat** [comment]
I think you might be correct, I already fixed few of these but it looks like something leaked through the cracks.

**Comment by aaltat** [comment]
Good catch and thank you for reporting the problem. This is bug was caused the type conversion enhancements in Robot Framework 4.0 and bug in the type hints of the keyword. Please note that with RF 4.0, you should be able to use `None` (as string) and `${None}` is not needed. 

This should be now fixed in the 5.1.2 release. Please test is and report back if you encounter new problems.

**Comment by bharkreader** [comment]
Wow, that was fast. You're awesome! Thanks!

Good tip on not needing ${None} too!
